subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -1235,9 +1233,17 @@ static void ticker_stop_sync_lost_op_cb(uint32_t status, void *param)
 
 static void sync_lost(void *param)
 {
-	struct ll_sync_set *sync = param;
+	struct ll_sync_set *sync;
 	struct node_rx_pdu *rx;
 
+	/* sync established was not generated yet, no free node rx */
```

Comment:
Is this the only one condition that the `sync->lll_sync_prepare` may differ from `lll_sync_prepare`, I mean no sync established event due to lack of free `node_rx`?

Reply:
yes, `sync->lll_sync_prepare` is set to `lll_sync_prepare` when ULL generates the sync established event towards HCI here https://github.com/zephyrproject-rtos/zephyr/pull/42572/files#diff-05ca44e3fbcc4f5bd0abd6e72153924632b01ba62979cf73b118d92cc3c836c7R833